source/_patterns/04-pages/grid-examples/single-column-partial/single-column-partial.twig

```diff
@@ -0,0 +1,43 @@
+{% set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['grid', 'grid--inside-elife']) }) -%}
```

Comment:
Currently using the CSS class `grid--inside-elife` as this grid is replicating that used on Inside eLife articles (see [an example](https://elifesciences.org/inside-elife/bd416611/webinar-report-graphic-design-tips-for-creating-effective-figures). Obviously this class name will need changing.